### PR TITLE
update polkadot libs

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

naïve attempt to update all polkadot deps to latest.

### ❓ Context

- **Impacted projects**: LLD, LLM 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** bot + integration https://github.com/LedgerHQ/ledger-live/actions/runs/3152938270
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/release-notes.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+    "tag_name": "2.48.0",
+    "body": "\n### 🚀 Features\n\n- Introducing a new way to swap crypto — **compare quotes from different platforms and choose the best offer**. Select the coin you want to swap and explore providers, KYC requirements, exchange rates, and more. In addition, you can now fully leverage DeFi protocols to swap ERC-20 tokens on decentralised exchanges.\n- Olá! From today, **Ledger Live supports Brazilian Portuguese** 🇧🇷 You can change the language in the settings menu under General → Display language.\n- When sending BTC, a **warning message** will alert you if there are any **pending incoming transactions**.\n\n### 🐛 Fixes\n\nWe’ve tidied up some bugs that caused issues when sending Zcash and signing transactions. Everything is running smoothly again.\n"
   }
-]
+]
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/release-notes.json:6]: Shouldn't belong in this PR, right?

File: libs/ledger-live-common/package.json [hunk 2]
```diff
@@ -146,11 +146,11 @@
...
-    "@polkadot/types": "8.12.2",
-    "@polkadot/types-known": "8.12.2",
-    "@polkadot/util": "10.0.2",
-    "@polkadot/util-crypto": "10.0.2",
-    "@polkadot/wasm-crypto": "6.0.1",
+    "@polkadot/types": "9.4.2",
+    "@polkadot/types-known": "9.4.2",
+    "@polkadot/util": "10.1.9",
+    "@polkadot/util-crypto": "10.1.9",
+    "@polkadot/wasm-crypto": "6.3.1",
```
Comment (haammar-ledger) [inline libs/ledger-live-common/package.json:153]: There was an issue with wasm-crypto 6.2.3 unfortunately.
https://ledgerhq.atlassian.net/browse/LIVE-2976
I have little hope it works now, but let's test again...

**Comment by gre** [comment]
i'm going to close the PR for now, it's easy for anyone from blockchain team to retry this & takeover 👍 , possibly without the wasm update part, it seems the bot was able to run correctly as well as the test regression.
unfortunately we don't have yet a way to check in LLM environment.
